### PR TITLE
fix: apply isObjectEqual to dashboard widget info helper

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [x] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

add isObjectEqual to core-lib which considers undefined property and absent property are the same.

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: packages/core-lib/src/index.ts [hunk 1]
```diff
@@ -66,6 +68,18 @@ export const isNotEmpty = (value): boolean => {
     return !isEmpty(value); // String, Object
 };
 
+export const isObjectEqual = (objValue: object, othValue: object) => {
```
Comment (sulmoJ) [inline packages/core-lib/src/index.ts:71]: 👍 👍 👍 👍

File: packages/core-lib/src/index.ts [hunk 2]
```diff
@@ -66,6 +68,18 @@ export const isNotEmpty = (value): boolean => {
...
+    const _isEqual = isEqual(objValue, othValue);
+    if (_isEqual) return true;
+    const keys = union(Object.keys(objValue), Object.keys(othValue));
+    return isEqualWith(objValue, othValue, (a, b) => keys.every((key) => {
+        if (typeof a[key] === 'object' && typeof b[key] === 'object') {
+            return isObjectEqual(a[key], b[key]);
+        }
+        return isEqual(a[key], b[key]);
+    }));
```
Comment (sulmoJ) [inline packages/core-lib/src/index.ts:80]: question: Can I think of it as 'deepEqual'?

**Review by sulmoJ — APPROVED** [review]
COOL

File: packages/core-lib/src/index.ts (hunk 2 above)
Comment (WANZARGEN) [reply packages/core-lib/src/index.ts:80]: yes, like the isEqual.

**Pull request merged**